timesketch/ui/css/ts.scss

```diff
@@ -230,11 +230,19 @@ header {
     height:34px;
     border:1px solid #d1d1d1;
     width:150px;
+    outline: none;
 }
 
 .select-view {
     margin-left:5px;
-    outline: none;
+}
+
+.select-graph-view {
+  height:57px;
+  background-color:#f9f9f9;
+  -webkit-border-radius: 0px;
```

Comment:
Please write `border-radius` here and use autoprefixer to append prefixes for all browsers.

Reply:
https://github.com/postcss/postcss-loader  <- use this

Reply:
Sounds good, will do in a followup CL.